src/component/filterButton/ascButton/AscButton.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Button } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+function AscButton(props) {
```

Comment:
this is wrong pattern. 
we making duplicates of the same button.

this is also wrong. if not clear ask me @saurabhketech

Reply:
@imsandip2085 this is wrong i already explained you need use one component for each button will be single component for all pages